pytorch_lightning/trainer/trainer.py

```diff
@@ -494,7 +494,9 @@ def __init__(
 
         # how much of the data to use
         # TODO: remove in 0.10.0
-        if overfit_pct > 0:
+        if overfit_pct is not None:
+            rank_zero_warn("Argument `overfit_pct` is now set by `overfit_batches` since v0.8.0"
+                           " and this argument will be removed in v0.10.0", DeprecationWarning)
```

Comment:
1.0.0
```suggestion
                           " and this argument will be removed in v1.0.0", DeprecationWarning)
```

Reply:
I would rather keep smaller granularity and if finally goes with 1.0 right after 0.9 we id very trivial to do all these cleaning compare to having everything as 1.0 and no ide when it was really meant to replace...